studier/faq/body.md

```diff
@@ -1,3 +1,3 @@
 
 # Vanliga frågor
-Svar på vanliga frågor går att hitta på [THS hemsida](https://ths.kth.se/en/help)
```

Comment:
Do we have an English FAQ page?

Reply:
But I think a case can be made for removing both the Swedish and English versions of our FAQ pages as they're only glorified links to THS in their current states. Perhaps the Study Board can create actual FAQ:s relevant to our chapter in the future.

Reply:
Did not see the comment